src/admin/components/SearchForm.scss

```diff
@@ -0,0 +1,28 @@
+.AdminSearchForm-form {
+  display: flex;
+
+  .AdminSearchForm-query {
+    background: url('~amo/img/icons/magnifying-glass.svg') 18px 50% no-repeat;
+    background-size: 26px;
```

Comment:
Ah, right, I'll check it... maybe it's the same icon and I'll just restore it :)